### PR TITLE
Client UI activity

### PR DESCRIPTION
**Pull request by Enkhzul001**
I added password match, arranged the order of the forms, and made gender have dropdown selection feature.

### CODE REVIEW

File: app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java [hunk 1]
```diff
@@ -1,15 +1,133 @@
...
+    TextView passwordAlert, confirmAlert, emailAlert, userAlert, firstAlert, lastAlert, homeAlert, phoneAlert;
+    private Button buttonsignup;
+
+    @SuppressLint("MissingInflatedId")
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_client_registration);
+        //find password, confirm password, name, email id, home address, phone, signup button
+        user = findViewById(R.id.username);
```
Comment (yuanshu777) [inline app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java:34]: for this, we can use data binding to make our code cleaner

Comment (yuanshu777) [reply app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java:34]: I'm doing this

**Review by yuanshu777 — APPROVED** [review]
Everything looks great. It's just that we can use data binding to make the code easier to track. I'M working on this right now

File: app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java (hunk 1 above)
Comment (Enkhzul001) [reply app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java:34]: Oh I see. Yeah that would make things cleaner. Let me just make a new commit.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">JH Mental Helper</string>
+    <string-array name="genders">
+        <item>Gender</item>
+        <item>Female</item>
```
Comment (Lam207) [inline app/src/main/res/values/strings.xml:5]: I guess you could add `Unmentioned` field here, too, after my PR is approved. Yeah sorry, the updated version of `Gender` is still pending on request.

**Pull request closed without merging**